src/scss/trumps/_spacing.scss

```diff
@@ -0,0 +1,73 @@
+@use '../tools/mixins/utilities';
+
+
+@mixin trumps-spacing() {
```

Comment:
I wonder if it's worth putting what you wrote in the PR description in a comment at the top of this file (to act as documentation for this?)